From 4114e1d3bb4298644e1153426a8d5e3b6bf9adda Mon Sep 17 00:00:00 2001
From: Yanjiang Jin <yanjiang.jin@windriver.com>
Date: Wed, 19 Oct 2016 10:47:20 +0000
Subject: [PATCH 425/666] fsl_dpa_offload: ipsec: rename addr_match() to
 dpa_addr_match()

include/net/xfrm.h has a same name function.

Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/staging/fsl_dpa_offload/dpa_ipsec.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/dpa_ipsec.c b/drivers/staging/fsl_dpa_offload/dpa_ipsec.c
index 29e8772..fd3faa01 100644
--- a/drivers/staging/fsl_dpa_offload/dpa_ipsec.c
+++ b/drivers/staging/fsl_dpa_offload/dpa_ipsec.c
@@ -3006,7 +3006,7 @@ static int store_policy_param_to_sa_pol_list(struct dpa_ipsec_sa *sa,
 	return 0;
 }
 
-static inline int addr_match(struct dpa_offload_ip_address *addr1,
+static inline int dpa_addr_match(struct dpa_offload_ip_address *addr1,
 			     struct dpa_offload_ip_address *addr2)
 {
 	if (addr1->version != addr2->version)
@@ -3060,8 +3060,8 @@ static int find_policy(struct dpa_ipsec_sa *sa,
 
 		if (cpol->dest_prefix_len != pol->dest_prefix_len ||
 		    cpol->src_prefix_len != pol->src_prefix_len ||
-		    !addr_match(&cpol->dest_addr, &pol->dest_addr) ||
-		    !addr_match(&cpol->src_addr, &pol->src_addr) ||
+		    !dpa_addr_match(&cpol->dest_addr, &pol->dest_addr) ||
+		    !dpa_addr_match(&cpol->src_addr, &pol->src_addr) ||
 		    cpol->protocol != pol->protocol ||
 		    cpol->masked_proto != pol->masked_proto ||
 		    cpol->priority != pol->priority)
-- 
2.7.4

